### ISSUE TITLE
recaptcha v3 does not work in MSIE 11

### ISSUE THREAD

**Issue by jirihradil** [ambethia/recaptcha#332]
Hi, 

we're facing an issue with recaptcha v3 in Microsoft Internet Explorer 11 (default configuration, no plugins, clean installation with Windows 10, version 11.615.17763.0). 

Recaptcha token is not fetched because of JS errors/double rendered ending script tag - please see the screenshots attached.

We use common helper attributes, not playing with JS generation/callbacks/scripts etc.

```
<%= recaptcha_v3(action: 'login', hl: I18n.locale, site_key: 'oursitekey') %>
```

In all other browsers (Chrome, Opera, Firefox) we don't have any issues at all. Unfortunately, a lot of our clients still use MSIE and we cannot switch them.

I didn't find any information that reCaptcha v3 should not work in MSIE 11, so could it be a gem's JS script issue?

Anyway, thanks for the awesome gem!

![Screenshot (3)](https://user-images.githubusercontent.com/656029/63754656-8a833080-c8b5-11e9-9ce3-775181e3f93f.png)
![Screenshot (4)](https://user-images.githubusercontent.com/656029/63754653-89ea9a00-c8b5-11e9-93aa-918f01946835.png)
![Screenshot (5)](https://user-images.githubusercontent.com/656029/63754654-89ea9a00-c8b5-11e9-9d1c-618350931086.png)

**Comment by grosser** [comment]
Can you  `bundle open recaptcha` and delete js until the error goes away ?
... the double closing </script> looks weird ...

**Comment by jirihradil** [comment]
Sorry, don't get it. Yes, I can delete JS in the source code which causes the problem but then the captcha will stop working, won't it? :) I tried it again in the simplest possible example and I can say that the gem is not working in MSIE 11 :(

**Comment by grosser** [comment]
Delete source code until the error goes away, to see what part exactly
causes the error ... then try to improve that part.

**Comment by jirihradil** [comment]
I "improved that part" by implementing captcha v3 manually because I found the current code over-complicated. Here is how to do that:

https://dev.to/morinoko/adding-recaptcha-v3-to-a-rails-app-without-a-gem-46jj

The code from the link (which is just a follow-up of Google reCAPTCHA docs) works perfectly in MSIE.

**Comment by grosser** [comment]
looks good! :)

**Comment by thomasbrus** [comment]
This has to do with `async` not being supported in IE11. I [created a fork](https://github.com/bookingexperts/recaptcha) that should fix this.

**Comment by grosser** [comment]
with IE changing to chromium, this should be fixed soon, right ?

**Comment by thomasbrus** [comment]
@grosser I don't think so 😅 IE11 is no longer being updated afaik. `async/await` already works in newer versions of IE/Edge though (see https://caniuse.com/#search=async)

**Comment by grosser** [comment]
good to know :)
